tests/static-analysis/scanleft.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+include __DIR__ . '/../../vendor/autoload.php';
+
+use loophp\collection\Collection;
+use loophp\collection\Contract\Collection as CollectionInterface;
+
+$sum = static fn (int $carry, int $value): int => $carry + $value;
+$concat = static fn (?string $carry, string $string): string => sprintf('%s%s', (string) $carry, $string);
+
+/**
+ * @param CollectionInterface<int, int> $collection
+ */
+function scanleft_checkListInt(CollectionInterface $collection): void
```

Comment:
```suggestion
function scanLeft_checkListInt(CollectionInterface $collection): void
```

same with the filename

Reply:
and in the other one

Reply:
Done!